node/src/command.rs

```diff
@@ -164,9 +164,21 @@ fn extract_genesis_wasm(chain_spec: &Box<dyn sc_service::ChainSpec>) -> Result<V
 pub fn run() -> Result<()> {
 	let cli = Cli::from_args();
 	match &cli.subcommand {
-		Some(Subcommand::BuildSpec(cmd)) => {
-			let runner = cli.create_runner(cmd)?;
-			runner.sync_run(|config| cmd.run(config.chain_spec, config.network))
+		Some(Subcommand::BuildSpec(params)) => {
+			let runner = cli.create_runner(&params.base)?;
+			runner.sync_run(|config| {
+				if params.mnemonic.is_some() || params.accounts.is_some() {
+					params.base.run(
+						Box::new(chain_spec::development_chain_spec(
+							params.mnemonic.clone(),
+							params.accounts,
+						)),
+						config.network,
+					)
+				} else {
+					params.base.run(config.chain_spec, config.network)
+				}
```

Comment:
Okay, I now understand that the new flags take precedence over a `--chain` argument. This is fine, but we need to communicate it to users clearly. If structopt allows it, we should make the new flags "conflict" with `--chain`. If structopt doesn't support that, then we should print a message like:
```
Warning: You've supplied both `--mnemonic` and `--chain`.
Your `--chain` parameter will be ignored.
Creating a development spec with prefunded accounts.
```

Reply:
it actually supports it :+1:

Reply:
How? The snippet quoted above calls `development_chain_spec` on line 172. What do you expect will happen if you run

```
moonbeam build-spec --chain alphanet --mnemonic "bottom drive ..."
```

Reply:
With the latest changes I mean. I added `conflicts_with=chain ` in the structopt and now if you run that you get `error: The argument '--chain <CHAIN_SPEC>' cannot be used with '--accounts <accounts>'`

Reply:
same with --mnemonics and --chain

Reply:
Oh, I see! That's excellent!